Server/conf/oxauth-config.json

```diff
@@ -336,5 +336,5 @@
     "deviceAuthzRequestExpiresIn": 1800,
     "deviceAuthzTokenPollInterval": 5,
     "deviceAuthzResponseTypeToProcessAuthz": "code",
-    "return200OnClientRegistration": true
```

Comment:
Lets not remove `"return200OnClientRegistration": true` which was added in previous PR.

Reply:
Yes, I hadn't noticed, it's fixed now.